src/utils/initialize-git-repo.js

```diff
@@ -14,11 +14,22 @@
 
 const runCommand = require('./run-command');
 
-const initializeGitRepo = async (repoPath) => {
+const initializeGitRepo = async (repoPath, templateValues) => {
+  console.log('Initialize git repo');
```

Comment:
we should eslint ignore these

Reply:
Can I just globally disable that linting rule. This is a cli tool, It's going to be rife with console logs.

Reply:
Addressed in b9d4159 by ignoring this rule globally